noetic/distribution.yaml

```diff
@@ -11082,6 +11082,11 @@ repositories:
       type: git
       url: https://github.com/rail-berkeley/serl_franka_controllers.git
       version: main
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/rail-berkeley/serl_franka_controllers.git
```

Comment:
I'd recommend instead a separate release repository.  The extra branches and tags are used by bloom to keep track of releases and other metadata.

Here's an example release repo : https://github.com/ros-gbp/xacro-release

For ROS 1 the instructions on creating a release repository are here: https://wiki.ros.org/bloom/Tutorials/FirstTimeRelease#Creating_a_Release_Repository

If you ever encounter an issue where bloom cant create the pull request (sometimes a random network failure, but [usually it's bloom either not having a token or the token having expired](https://wiki.ros.org/bloom/Tutorials/GithubManualAuthorization)), then please use the `--pull-request-only` option to `bloom-release` after resolving the issue.